src/dev/run_i18n_check.js

```diff
@@ -17,13 +17,19 @@
  * under the License.
  */
 
+import chalk from 'chalk';
 import { run } from './run';
 import { extractDefaultTranslations } from './i18n/extract_default_translations';
 
 run(async ({ flags: { path, output, 'output-format': outputFormat } }) => {
-  await extractDefaultTranslations({
-    paths: Array.isArray(path) ? path : [path || './'],
-    output,
-    outputFormat,
-  });
+  try {
+    await extractDefaultTranslations({
+      paths: Array.isArray(path) ? path : [path || './'],
+      output,
+      outputFormat,
+    });
+  } catch (e) {
+    console.error(`${chalk.white.bgRed(' I18N ERROR ')} ${e.message || e}`);
```

Comment:
The `run()` function implements standardized failure handling, please don't override it as removing the stacktrace from expected errors can make them really hard to debug. If you have specific errors that don't require a stack trace please throw a `FailError` instead, like so:

```
import { createFailError } from '../run';
// ...
throw createFailError(`${chalk.white.bgRed(' I18N ERROR ')} some error message`);
```

`run()` will only print the error message from `FailError`s and will print the stacktrace for other unexpected errors.